transform-plugins/docs/NullFieldSplitter-splittertransform.md

```diff
@@ -3,62 +3,63 @@
 
 Description
 -----------
-This plugin is used when you want to split records based on whether a specific field is null or not.
-Records with a null value for the field are sent to the ``null`` port while records with a non-null
-value are sent to the ``nonnull`` port.
+Null Field Splitter is a Guavus Enterprise Accelerator that is used when you want to split records based on whether a specific field in a record is null or not.
+
+The records containing fields with a null value are sent to the ``null`` port while records with non-null value fields are sent to the ``nonnull`` port.
+
+Use Case
+-----------
+
+Consider a scenario wherein you have a large number of records and you want to separate records with null values from the ones with non-null values. This can be done by configuring the accelerator as explained in the following section.
 
 Properties
 ----------
-**field:** Specifies which field should be checked for null values. (Macro-enabled)
```

Comment:
add "generate schema" button . which is responsible to genrate output schema on the basis of configuration .